mysite/polls/views.py

```diff
@@ -0,0 +1,43 @@
+from django.shortcuts import render
+import search
+import json
+from django.http import JsonResponse
+from django.http import HttpResponse
+import re
+# Create your views here.
+
+
+def make_link_for_html(movie_id):
+    link = '<a href = "http://localhost:8000/polls/info/?q=' \
+            + movie_id + '"> Link </a>'
+    return link
+
+
+def make_link(movie_id):
+    link = 'http://localhost:8000/polls/info/?q=' \
+            + movie_id  
+    return link
+
+
+def index(request):
+    ans = request.GET.get('q', 'Nothing')
+    movie_ids, movie_names, id_name_map = search.get_movie_names(ans)
+    print movie_ids
```

Comment:
you shouldn't leave debugging statements in your PR. if you want to log something, then you can the logging module - https://docs.python.org/2/library/logging.html